default.py

```diff
@@ -111,7 +114,7 @@ def build_url(query):
 
 # Launch
 elif mode[0] == 'folder':
-    lutris = settings.getSetting('lutris_executable')
+    lutris = executable
```

Comment:
Don't think executable exists in this scope when you're launching the game. May need to bring some of that logic into its own function.